src/main/java/de/dennisguse/opentracks/stats/TrackStatisticsUpdater.java

```diff
@@ -41,23 +40,15 @@
  */
 public class TrackStatisticsUpdater {
 
-    /**
-     * The number of speed reading to smooth to get a somewhat accurate signal.
-     */
-    @VisibleForTesting
-    private static final int SPEED_SMOOTHING_FACTOR = 25;
-
     private static final String TAG = TrackStatisticsUpdater.class.getSimpleName();
     /**
      * Ignore any acceleration faster than this.
      * Will ignore any speeds that imply acceleration greater than 2g's
-     * 2g = 19.6 m/s^2 = 0.0002 m/ms^2 = 0.02 m/(m*ms)
      */
-    private static final double MAX_ACCELERATION = 0.02;
+    private static final double SPEED_MAX_ACCELERATION = 2 * 9.81;
```

Comment:
Where 9.81 value comes from? Why not to use 19.6 value directly?

Reply:
9.81 is one G (gravity earth).
It is just documenting that it is used.